api/pkg/transformer/types/table/table.go

```diff
@@ -218,18 +218,60 @@ func (t *Table) Sort(sortRules []*spec.SortColumnRule) error {
 
 // UpdateColumnsRaw add or update existing column with values specified in columnValues map
 func (t *Table) UpdateColumnsRaw(columnValues map[string]interface{}) error {
-	df := t.DataFrame()
+	origColumns := t.Columns()
+	updateCol := map[string]bool{} //name of col that are updates
+	combinedColumns := make([]*series.Series, 0)
+
+	columnValues = broadcastScalar(columnValues, t.NRow())
```

Comment:
Why perform `broadcastScalar` ?

Reply:
It is to make all columns have the same number of rows, like how it is done in createSeries function below. Otherwise may have dimensions different error.